### PR TITLE
[Installers] Added RDS Exporter

### PR DESCRIPTION
**Pull request by aditya-opsverse**
Tested this:
 1. By setting up a test RDS on AWS `dev` cluster(`database-1`).
 2. By setting up a test instance on GCP (`aditya-rds-test`).
 3. RDS Dashboard link - https://opsverse-venus.observe.devopsnow.cloud/d/BfU3KE44z/aws-rds-dashboard?orgId=1

### CODE REVIEW

**Review by vinaya-devopsnow — APPROVED** [review]

File: prometheus-exporters/install-exporter-amd64.sh [hunk 1]
```diff
@@ -816,6 +865,21 @@ EOF
...
+
+  if [ "$EXPORTER" == "rds" ]; then
+    cat << EOF > /etc/opsverse/targets/${EXPORTER}-exporter.json
+[
+  {
+    "labels": {
+      "job": "integrations/rds-exporter"
+    },
+    "targets": [
+      "localhost:9042"
```
Comment (arul-opsverse) [inline prometheus-exporters/install-exporter-amd64.sh:879]: @aditya-opsverse, I thought that the scrape path was something other than the default `metrics` ? If so, we may have to adjust this `json`.

File: prometheus-exporters/install-exporter-amd64.sh [hunk 2]
```diff
@@ -385,6 +401,18 @@ DATA_SOURCE_NAME="postgresql://postgres:postgres123@localhost:5432/?sslmode=disa
 EOF
   fi
 
+  if [ "$EXPORTER" == "rds" ]; then
+    cat << EOF > /etc/opsverse/exporters/rds/config.yaml
+instances:
+  - region: <region>
+    instance: <instance>
+    aws_access_key: <aws_access_key>
+    aws_secret_key: <aws_secret_key>
```
Comment (arul-opsverse) [inline prometheus-exporters/install-exporter-amd64.sh:410]: The user will need to edit this file and then restart the `rds-exporter` service. 

Can we please add a `doc` page for configuring this exporter?

Comment (aditya-opsverse) [reply prometheus-exporters/install-exporter-amd64.sh:410]: Yeah Arul
As this was the high priority for Snapmint, I thought I would add docs later.

Comment (aditya-opsverse) [reply prometheus-exporters/install-exporter-amd64.sh:410]: I will work on doc page.

File: prometheus-exporters/install-exporter-amd64.sh (hunk 1 above)
Comment (aditya-opsverse) [reply prometheus-exporters/install-exporter-amd64.sh:879]: I didn't quite get this Arul.

**Pull request merged**